docs/Index.tsx

```diff
@@ -0,0 +1,371 @@
+import {Model as BackboneModel} from 'backbone';
```

Comment:
Yes, that file is a big blob of code. For now, we only need a place where to put code to test our components, but we will eventually created a structured project for component doc.